src/lib/registration/sw_mgmt.rb

```diff
@@ -551,19 +569,19 @@ def self.use_credentials(filename)
     def self.find_addon_updates(addons)
       log.info "Available addons: #{addons.map(&:identifier)}"
 
-      products = Pkg.ResolvableProperties("", :product, "")
+      products = Y2Packager::Resolvable.find(kind: :product)
 
       installed_addons = products.select do |product|
-        (product["status"] == :installed || product["status"] == :removed) &&
-          product["type"] != "base"
+        (product.status == :installed || product.status == :removed) &&
+          product.type != "base"
       end
 
-      product_names = installed_addons.map { |a| "#{a["name"]}-#{a["version"]}-#{a["release"]}" }
```

Comment:
Why are you dropping the "release" part?

Reply:
As far I am remembering Lade has told me that there is no "release" method available. Up to now it has not hurt us because it is a hash.

Reply:
Ok, fine with me. I just wanted to know the reason. Thanks.